app.js

```diff
@@ -0,0 +1,61 @@
+$(document).ready(function(event){
+/////Declare variables/////////	
+	var tastes=['fruity', 'bitter', 'sweet', 'salty', 'strong']
+	var questions=[];
+	var ingredients=[];
+	var pantry=[]; 
+	var preferences={}; 
+	var questionCounter = 0; 
+
+////////Question Constructor/////// 
+	var question = function(taste) {
+		this.taste=taste;
+		this.question='Do ye like ' + taste + ' drinks?'; 
+	}
+////////Make Questions///////////
+	for (var i=0; i<tastes.length; i++) {
+	questions[i] = new question(tastes[i]);
+	}
+
+//////Ask Questions///////////	
+	function askQuestions() {
+		if (questionCounter<questions.length) {
+			$('.ask-question').removeClass('hidden').prepend('<h3 class="question">' + questions[questionCounter].question + '</h3>'); 
+		} else {
+			$('.ask-question').remove();
+			$('main').append('<p>Thanks for reviewing. Now I need to build the rest of the app!</p>')
+		}	
+	}
+
+///////Record Answers////////////
+	function recordAnswer(taste, preference) {
+		preferences[taste] = preference;
+	}
+
```

Comment:
Consider standardizing white space between functions. This one has 2 lines, other functions have 1